### PR TITLE
New package: elog-3.1.4.1

### PR DESCRIPTION
**Pull request by the-wrench99**

### CODE REVIEW

**Comment by Vaelatern** [comment]
https://github.com/void-linux/void-packages/pull/4782
@the-wrench99 `git push -f` is your friend, you don't need to close and re-open PRs.

File: srcpkgs/elog/template [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+homepage="https://elog.psi.ch/elog/"
+distfiles="https://elog.psi.ch/elog/download/tar/$pkgname-$_distver.tar.gz"
+checksum=4bfa2bef6cfa20aba73c9c6c044a4c83f20cf03e7de3c0dda4a4477f28b7b72e
+
+do_build() {
+	make CC="$CC" LD="$LD" AR="$AR" \
+			RANLIB="$RANLIB" OBJDUMP="$OBJDUMP" \
+			CFLAGS=" -Imxml $CFLAGS" LDFLAGS="$LDFLAGS" \
+			all
+}
```
Comment (Vaelatern) [inline srcpkgs/elog/template]: Why are you overwriting this at all?

Comment (the-wrench99) [reply srcpkgs/elog/template]: There needs to be "-Imxml" set in the CFLAG variable and I did not find an option to do so in the gnu_makefile build style

Comment (maxice8) [reply srcpkgs/elog/template]: just add CFLAGS="-lmxml" to the top-level of the template

**Pull request merged**